### PR TITLE
Fixed McCabe code for Pylama

### PR DESCRIPTION
**Pull request by apuignav**
Seems the code is C901

```shell
[0:48]~/scratch/tools-hepdata[master]$ pylama -lmccabe wizard.py
wizard.py:11:1: C901 'ask_question' is too complex (13) [mccabe]
[0:49]~/scratch/tools-hepdata[master]$ pylama --version
pylama 7.3.1
```

### CODE REVIEW

**Comment by apuignav** [comment]
A further question: shouldn't McCabe be 'I' to be coherent with the Pylint entry?

File: autoload/neomake/makers/ft/python.vim [hunk 1]
```diff
@@ -72,17 +72,8 @@ function! neomake#makers#ft#python#flake8() abort
...
-            if a:entry.nr == 407
-                let type = 'E'  " 'an undefined __future__ feature name was imported'
-            else
-                let type = 'W'
-            endif
-        else
-            let type = 'E'
-        endif
+    if a:entry.type ==# 'F'  " PyFlake errors
+        let type = 'E'
```
Comment (blueyed) [inline autoload/neomake/makers/ft/python.vim]: You should not revert this?!

File: autoload/neomake/makers/ft/python.vim [hunk 2]
```diff
@@ -152,11 +143,12 @@ function! neomake#makers#ft#python#pep257() abort
 endfunction
 
 function! neomake#makers#ft#python#PylamaEntryProcess(entry) abort
+    echom a:entry.text
```
Comment (blueyed) [inline autoload/neomake/makers/ft/python.vim]: Left-over.

File: autoload/neomake/makers/ft/python.vim [hunk 3]
```diff
@@ -152,11 +143,12 @@ function! neomake#makers#ft#python#pep257() abort
...
     if a:entry.type ==# 'C' && a:entry.text =~# '\v\[%(pycodestyle|pep8)\]$'
         call neomake#makers#ft#python#Pep8EntryProcess(a:entry)
     elseif a:entry.type ==# 'D'  " pydocstyle/pep257
         let a:entry.type = 'W'
-    elseif a:entry.type ==# 'E901'  " mccabe
+    elseif a:entry.text =~# '^C901'  " mccabe
```
Comment (blueyed) [inline autoload/neomake/makers/ft/python.vim]: What about `type ==# 'C' && `nr == 901`?  (but feel free to keep it like it is now).

**Comment by blueyed** [comment]
> A further question: shouldn't McCabe be 'I' to be coherent with the Pylint entry?

Makes sense.

**Comment by apuignav** [comment]
Sorry! That was sloppy. All changes implemented.

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**